assets/src/application/ui/display/TimezoneTimeInfo/Trigger.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { forwardRef } from 'react';
+import { IconButton } from '@chakra-ui/core';
```

Comment:
need to import an adapter from the infrastructure layer

Reply:
created adapter in `@infraUI/inputs`